birdsong/templates/tags/subscribe.html

```diff
@@ -0,0 +1,99 @@
+<form id="{{ BIRDSONG_SUBSCRIBE_FORM_ID }}" action="{{ form_action }}" method="post" class="needs-validation" {% if BIRDSONG_SUBSCRIBE_FORM_NOVALIDATE %}novalidate{% endif %}>
```

Comment:
remove the need-validation, if someone is using bootstrap they should override this template

Reply:
I'll have a look at replacing the bootstrap5 related code with something more generic, perhaps some raw html+css driven feedback.

I've opted to go with bootstrap styling because frontend html isn't something I'm particularly good at but I'll come up with something more standalone and perhaps even split it all up into separate html, js and css files. Is that the right direction?